src/main/java/page_object/BasePage.java

```diff
@@ -1,92 +1,52 @@
-    private By tabTicketPrice = By.cssSelector("li a[href*='/Page/TrainPriceListPage.cshtml']");
-    private By tabLogout = By.cssSelector("a[href$='Logout']");
-    private By tabContact = By.cssSelector("a[href*='Contact']");
-    private By tabChangePassword = By.cssSelector("li a[href='/Account/ChangePassword.cshtml']");
-
-    //element
-    private WebElement getTabLogin() {
-        return DriverHelper.getDriver().findElement(tabLogin);
-    }
-
-    private WebElement getTabRegister() {
-        return DriverHelper.getDriver().findElement(tabRegister);
-    }
-
-    private WebElement getTabBookTicket() {
-        return DriverHelper.getDriver().findElement(tabBookTicket);
-    }
-
-    private WebElement getTabTimetable() {
-        return DriverHelper.getDriver().findElement(tabTimetable);
-    }
-
-    private WebElement getTabTicketPrice() {
-        return DriverHelper.getDriver().findElement(tabTicketPrice);
-    }
-
-    private WebElement getTabLogout() {
-        return DriverHelper.getDriver().findElement(tabLogout);
-    }
-
-    private WebElement getTabContact() {
-        return DriverHelper.getDriver().findElement(tabContact);
-    }
-
-    private WebElement getTabChangePassword() {
-        return DriverHelper.getDriver().findElement(tabChangePassword);
-    }
+    private final Link loginTab = new Link("css=a[href$='Login.cshtml']");
+    private final Link bookTicketTab = new Link("css=a[href$='BookTicketPage.cshtml']");
+    private final Link logoutTab = new Link("css=a[href$='Logout']");
+    private final Link contactTab = new Link("css=a[href*='Contact']");
+    private final Link registerTab = new Link("css=a[href*='Register']");
+    private final Link changePasswordTab = new Link("css=a[href*='ChangePassword']");
+    private final Link myTicketTab = new Link("css=a[href*='ManageTicket']");
 
     //method
     public void clickLoginTab() {
-        getTabLogin().click();
+        loginTab.click();
     }
 
     public void clickRegisterTab() {
-        getTabRegister().click();
+        registerTab.click();
     }
 
     public void clickBookTicketTab() {
-        getTabBookTicket().click();
-    }
-
-    public void clickTimetableTab() {
-        getTabTimetable().click();
-    }
-
-    public void clickTicketPriceTab() {
-        getTabTicketPrice().click();
+        bookTicketTab.click();
     }
 
     public void clickLogoutTab() {
-        getTabLogout().click();
+        logoutTab.click();
     }
 
     public void clickContactTab() {
-        getTabContact().click();
+        contactTab.click();
     }
 
     public void clickChangePasswordTab() {
-        getTabChangePassword().click();
+        changePasswordTab.click();
+    }
+
+    public void clickTabMyTicket() {
+        myTicketTab.click();
     }
 
     public boolean isLoginTabDisplayed() {
-        return ElementHelper.isElementDisplayed(this.getTabLogin());
+        return loginTab.isExist();
```

Comment:
isExist or isVisible? Make sure that you get the difference while using two of them

Reply:
I changed to isVisible. Please check again.